src/lib/server/xlat_eval.c

```diff
@@ -285,23 +285,23 @@ static xlat_action_t xlat_eval_one_letter(TALLOC_CTX *ctx, fr_cursor_t *out, REQ
 
 	case 'c': /* Current epoch time seconds */
 	{
-		struct timeval now;
+		fr_time_t now;
 
-		gettimeofday(&now, NULL);
+		now = fr_time();
 
 		MEM(value = fr_value_box_alloc(ctx, FR_TYPE_UINT64, NULL, false));
-		value->datum.uint64 = (uint64_t)now.tv_sec;
+		value->datum.uint64 = (uint64_t)now;
 	}
 		break;
 
 	case 'C': /* Current epoch time microsecond component */
 	{
-		struct timeval now;
+		fr_time_t now;
 
-		gettimeofday(&now, NULL);
+		now = fr_time();
 
 		MEM(value = fr_value_box_alloc(ctx, FR_TYPE_UINT32, NULL, false));
-		value->datum.uint32 = (uint64_t)now.tv_usec;
+		value->datum.uint32 = (uint64_t)now;
```

Comment:
No, again, you've just returned timeofday in _NANOSECONDS_ when it's meant to be _MICROSECONDS_, call ``fr_time_to_usec``